search_request.go

```diff
@@ -165,6 +186,9 @@ func (r *SearchRequest) header() interface{} {
 	if r.expandWildcards != "" {
 		h["expand_wildcards"] = r.expandWildcards
 	}
+	if v := r.allowPartialSearchResults; v != nil {
```

Comment:
Are you missing the `maxConcurrentShardRequests` header here?

Reply:
it wasn't here https://github.com/olivere/elastic/blob/release-branch.v7/search_request.go

Reply:
Ah yeah, that is only in the URL params, so it should be fine.